packages/@aws-cdk/toolkit-lib/lib/api/refactoring/digest.ts

```diff
@@ -50,7 +50,13 @@ export function computeResourceDigests(template: CloudFormationTemplate): Record
       return [refTarget];
     }
     if ('DependsOn' in value) {
-      return [value.DependsOn];
+      const result = Object.values(value).flatMap(findDependencies);
```

Comment:
Bit strange we have to call this again, since we are already doing it in line 61. Can you not:

```ts
const result = [];
if ('DependsOn' in value) {
  if (Array.isArray(value.DependsOn)) {
    result.push(...value.DependsOn);
  } else {
    result.push(value.DependsOn);
  }
}
result.push(...Object.values(value).flatMap(findDependencies))
return result;
```

?

Reply:
Approved anyway - take it or leave it.